src/guidellm/benchmark/benchmark.py

```diff
@@ -817,7 +817,9 @@ def from_stats(
                     ],
                     iter_counts=[req.output_tokens for req in total_with_output_first],
                     first_iter_counts=[
-                        req.prompt_tokens for req in total_with_output_first
+                        # prompt tokens + first token
```

Comment:
Yeah, good catch, found this in the refactor as well. We'll need to subtract one from the output_tokens as well since we are counting the first token here